.github/workflows/build_and_test.yml

```diff
@@ -494,53 +495,12 @@ jobs:
         name: test-results-sparkr--8-${{ inputs.hadoop }}-hive2.3
         path: "**/target/test-reports/*.xml"
 
-  proto-breaking-changes-check-master:
+  breaking-changes-buf:
     needs: [precondition]
-    if: always() && fromJson(needs.precondition.outputs.required).proto-breaking-changes-check == 'true'
-    name: Spark Connect proto backwards compatibility check (master)
-    continue-on-error: true
+    if: always() && fromJson(needs.precondition.outputs.required).breaking-changes-buf == 'true'
+    # Change 'branch-3.4' to 'branch-3.5' in master branch after cutting branch-3.5 branch.
+    name: Breaking change detection with Buf (branch-3.4)
```

Comment:
This name, I got it from https://github.com/bufbuild/buf-breaking-action#buf-breaking-action README.md